lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -39,6 +39,7 @@ body {
             margin-left: 0;
             position: absolute;
             right: 0.1rem;
+            margin-right: -5px;
```

Comment:
Generally speaking, I would like to avoid negative margin values, but in this case, it may be the easiest way to achieve this

Reply:
I agree. Resolving this for now, but quickly testing alternatives as well.

Reply:
For example, displaying the button as a block element requires changes in two positions which seems more error prone across screen sizes. The following was achieved using only positive values:

1. display: block;
2. left: 87%;
3. top: 6.2rem;

<img width="1680" alt="image" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/956bb32a-8e03-4b7c-b175-8e5fa4e2f9a3">

but again the negative margin seems to be a more straight-forward change for this instance.